### PR TITLE
Rebuild on the new protos

### PR DESCRIPTION
**Pull request by tdeebswihart**
**What changed?**
This repo has been rebuilt on top of the proto annotations introduced in temporalio/api#319 and the proto changes in temporalio/api#320

**Why?**

**How did you test it?**
I rebuilt the UI on top of this and all tests pass.

**Potential risks**
None. This change will only impact the UI

**Notes**
This will not be merged until https://github.com/temporalio/api/pull/320 is in

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -24,7 +24,7 @@ PINNED_DEPENDENCIES := \
 
 
 PROTO_ROOT := proto/api
-PROTO_FILES = $(shell find $(PROTO_ROOT) -name "*.proto")
+PROTO_FILES = $(shell find $(PROTO_ROOT)/temporal -name "*.proto")
```
Comment (bergundy) [inline Makefile:27]: What's the reason for this change?

**Review by bergundy — APPROVED** [review]

**Comment by stephanos** [comment]
I'm a bit confused; I was under the impression that now that https://github.com/temporalio/api/pull/320 is merged, it would automatically create a commit in this repo. I [found that out last time](https://github.com/temporalio/api-go/pull/122#discussion_r1356708511) when I tried to make a change here and added this to the README:

> Note that any changes merged to [api](https://github.com/temporalio/api) will automatically trigger a [GitHub workflow](https://github.com/temporalio/api-go/blob/master/.github/workflows/update-proto.yml) that recompiles the proto files and commits the results to master.

**Comment by tdeebswihart** [comment]
> I'm a bit confused; I was under the impression that now that [temporalio/api#320](https://github.com/temporalio/api/pull/320) is merged, it would automatically create a commit in this repo. I [found that out last time](https://github.com/temporalio/api-go/pull/122#discussion_r1356708511) when I tried to make a change here and added this to the README:
> 
> > Note that any changes merged to [api](https://github.com/temporalio/api) will automatically trigger a [GitHub workflow](https://github.com/temporalio/api-go/blob/master/.github/workflows/update-proto.yml) that recompiles the proto files and commits the results to master.
> 
> The last one was [bb03061](https://github.com/temporalio/api-go/commit/bb03061759c82712a4b933f5175834baebee9c9a) and it updated the git submodule as expected.

It didn't seem to but it wouldn't have fixed the Makefile so a manual change is still necessary
